setup.py

```diff
@@ -60,19 +60,19 @@ def recursive_data_files(directory, install_directory):
     scripts = []
 
 dev_requires = [
-    "black",
-    "codespell",
+    "black==23.1.0",
```

Comment:
can you remind me why the pinning is needed here? I assume tox can use the requirements files right?
Is it because of renovate?